cmd/osv-scanner/scan/source/testdata/cassettes/TestCommand.yaml

```diff
@@ -1624,7 +1916,7 @@ interactions:
       status: 200 OK
       code: 200
       duration: 0s
-  - id: 15
+  - id: 16
```

Comment:
hmm this incremental id number is a bit annoying to have in diffs. With the custom marshaller and unmarshaller we can probably remove it with some effort. Something for a separate PR.